### ISSUE TITLE
CRC-Offload on KVM-Host

### ISSUE THREAD

**Issue by JochenKorge** [opnsense/core#5764]
Hi,
TL;DR:
you have to **enable** CRC-Checksum offload in new(er) versions of KVM-Virtualized OpnSenses.

Our setup consists of a 3 node Proxmox cluster hosting 2 OpnSense in HA.
We do use the qemu-guest-agent plugin and virtio NICs for the OpnSense-VMs.

After updating from 21.xx to 22.xx we got strange connectivity issues.
Connections that where routed by the OpnSense to and from the host and all VMs on that node that hosted the active opnsense where broken. All other nodes an VMs did work flawless. As did connections that wheren´t routed by the opnsense (e.g. connections within the same Subnet).
Migrating the FW or "Carp-Failover" to the Backup did move the unreachable hosts/VMs to the other Node instantly.
Wireshark told me that there where CRC-Checksum errors on TCP and UDP.

The following setting **fixed** it:
![grafik](https://user-images.githubusercontent.com/70751057/167845617-f81915c6-c40e-434a-893d-8909677a6dce.png)

This is **contrary** to the docs. (https://docs.opnsense.org/manual/virtuals.html)

Unfortunatly my Lab-Environment is out of order, so I dont want to post logs public (mail is ok though).
If I can be of any assistant let me know.

Cheers Jochen

**Comment by sempervictus** [comment]
Un-checking that box (and various combinations of all 3) did not help matters on our end, nor did any of the `vtnet`-specific sysctl settings in various flavors:
![image](https://user-images.githubusercontent.com/1331084/182971731-c3ee0f79-b51d-4714-b451-ca9db08a83b9.png)
22.1 and 22.7 cannot ping or otherwise communicate with any hosts in their local subnet _until_ running `service of onestop` at which point connectivity resumes... odd. Seems that there is some interaction between `vtnet` and `pf` which allows the device to route traffic but not communicate with hosts in its own local subnets.

**Comment by mimugmail** [comment]
Sounds you have upstream gateway in interface set
